web_src/js/features/repo-issue.js

```diff
@@ -119,14 +119,15 @@ export function initRepoIssueSidebarList() {
   $('.ui.dropdown.label-filter, .ui.dropdown.select-label').dropdown('setting', {'hideDividers': 'empty'}).dropdown('refreshItems');
 }
 
-function initIssueSearchDropdown(selector) {
+function initIssueSearchDropdown(selector, isDuplicateModal) {
```

Comment:
I don't think it is maintainable and have objection for doing so

Reply:
dace5ce, I'm not sure it is a better way, could you take a look?